CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changelog
 
   **deprecated** `CacheManager::tagResponse` in favor of `TagHandler::addTags`
 * **2015-05-08** Added configuration option for custom proxy client (#208)
+* **2015-05-08** Added support for a simple Etag header in the header configuration (#207)
```

Comment:
Just wondering: why do you add a date for each entry? Wouldn’t it make more sense to add one date to the release?